code/game/machinery/computer/apc_control.dm

```diff
@@ -193,12 +187,15 @@
 	interact(usr) //Refresh the UI after a filter changes
 
 /obj/machinery/computer/apc_control/emag_act(mob/user)
-	if(obj_flags & EMAGGED)
-		return
-	user.visible_message("<span class='warning'>You emag [src], disabling precise logging and allowing you to clear logs.</span>")
-	log_game("[key_name(user)] emagged [src] at [get_area(src)], disabling operator tracking.")
+	if(!authenticated) 
```

Comment:
This doesn't seem right...

Reply:
It's so they can log in without emagging it fully, that was the old behaviour.